pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -386,14 +391,22 @@ protected void addTableToH2(String tableName, Schema schema)
         h2FieldNamesAndTypes.toArray(new String[h2FieldNamesAndTypes.size()])) + ")").execute();
   }
 
+  private String getH2InsertQueryParam(String h2FieldNameAndType) {
+    String type = h2FieldNameAndType.split(" ")[1];
+    if ("json".equalsIgnoreCase(type)) {
+      return "? FORMAT JSON";
```

Comment:
This is needed for the comparison with Pinot rows. As per the [documentation](https://www.h2database.com/html/datatypes.html#json_type), this will save the data in `java.lang.String` format in H2 db.

Reply:
see my comments above. dont compare JSON results with H2. see how 
`JsonPathClusterIntegartionTest` is also not using H2 as comparison base